webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala

```diff
@@ -239,8 +239,10 @@ final case class ProjectsResponderADMLive(
   override def projectsGetRequestADM(): Task[ProjectsGetResponseADM] =
     projectService.findAll
       .flatMap(projects =>
-        if (projects.nonEmpty) { ZIO.succeed(ProjectsGetResponseADM(projects)) }
-        else { ZIO.fail(NotFoundException(s"No projects found")) }
+        if (projects.nonEmpty) {
+          val noSystemProjects: List[ProjectADM] = projects.filter(p => p.id.startsWith("http://rdfh.ch/projects/"))
+          ZIO.succeed(ProjectsGetResponseADM(noSystemProjects))
+        } else ZIO.fail(NotFoundException(s"No projects found"))
```

Comment:
not sure what the intended behaviour is: should `NotFoundException` be returned if there are no projects at all, or if there are system projects but no other projects?

Reply:
`NotFoundException` should be returned when there is no projects at all. System projects are supposed to be filtered out from the results anyway, so they shouldn't have been taken into the consideration here.